client/unit-tests/symbolic/test_atomics.py

```diff
@@ -0,0 +1,1111 @@
+import datetime
```

Comment:
can we change this file name to no longer be "test_atomics" now that you've changed the other file name?